examples/hosts/app-integration/external-data/src/model-interface/index.ts

```diff
@@ -154,4 +154,27 @@ export interface ITaskList extends IEventProvider<ITaskListEvents> {
 	// readonly handleExternalMessage: (message) => void;
 }
 
+/**
+ * Events emitted by {@link IBaseDocumentEvents}.
+ */
+export interface IBaseDocumentEvents extends IEvent {
+	/**
+	 * Emitted when task list collection has changed.
+	 */
+	(event: "taskListCollectionChanged", listener: () => void);
+}
+export interface IBaseDocumentInitialState {
```

Comment:
I'd just still like to see this named `ITaskListInitialState` instead at some point 😄 .